### DISCUSSION TITLE
Hardware recommendations) RockPro64, Odroid H3+

### DISCUSSION THREAD

**Discussion by salvobellino95** [blakeblackshear/frigate#5234] [General]
Hi! Does anyone has Frigate running on these boards?
Currently I have one RockPi 4c+ but I want to change device because it does not support hw acceleration in an easy way, the RockPro64 has newer linux kernels support so maybe it works. 
Odroid H3 is powered by an Intel CPU which supports QuickSync.

I will use only 4-6 cameras 

Thank you!

**Comment by timmyag** [comment]
I have a N6005 same CPU as the odriod on a different board no coral and .11 stugles, 200+ms Inference times, now running .12 b5 and Open Vino uses the GPU far better(though the model could be better) and im seeing 15-20ms times. Im running Proxmox, 2 docker hosts, Home assitant, Adgaurd, EMXQ, Z2MQTT, Frigate with 5 cams, CPU usage hovers about 30-35%.

**Comment by norbertjoni** [comment]
> Hi! Does anyone has Frigate running on these boards?
> 
> Currently I have one RockPi 4c+ but I want to change device because it does not support hw acceleration in an easy way, the RockPro64 has newer linux kernels support so maybe it works. 
> 
> Odroid H3 is powered by an Intel CPU which supports QuickSync.
> 
> 
> 
> I will use only 4-6 cameras 
> 
> 
> 
> Thank you!

Have you got your H3? Any feedback?

**Reply by salvobellino95** [reply]
No, I buyed a bmax b3 mini pc, works well with good price

**Comment by drejkus** [comment]
I have Frigate on a RockPro64. Running with custom ffmpeg build with hwaccel (v4l2_request), two cameras. Works well, but it kinda uses CPU even with HW decode, just slightly less than without it (but not by much, don't know why. Tests with BigBuckBunny MP4 give much bigger difference than my RTSP streams). So I would currently not use it for more than 3-4 cams, or use lower resolution than FHD.
